shiny/ui/_navs.py

```diff
@@ -985,18 +990,209 @@ def navset_pill_list(
     )
 
 
+class Default(Generic[T]):
+    def __init__(self, value: T):
+        self._default = value
```

Comment:
Is this getting used anywhere?

Reply:
No, I took this out but I must not have pushed the commit